### ISSUE TITLE
Double spaces

### ISSUE THREAD

**Issue by jer-sen** [dotansimha/graphql-code-generator#4141]
There are 2 spaces in Typescript types as `interface` (between `TypeName` and `extends`):

```
export interface TypeName  extends InterfaceName {
  readonly __typename?: 'TypeName';
  readonly type: PropertyType;
}
```

**Comment by dotansimha** [comment]
Thanks @jer-sen , this is a minor lint issue, and we usually don't fix those. 
It's recommended to ignore generated files from your linter/formatter.
 
I fixed this specific one in: https://github.com/dotansimha/graphql-code-generator/commit/247a8b0b5bc79b8aff08f60ba337c9f6892a4b16

**Comment by jer-sen** [comment]
Thanks @dotansimha

**Comment by dotansimha** [comment]
Fixed in `v1.15.2`
